consensus/src/dag/tests/order_rule_tests.rs

```diff
@@ -87,24 +84,23 @@ impl OrderedNotifier for TestNotifier {
         &self,
         ordered_nodes: Vec<Arc<CertifiedNode>>,
         _failed_authors: Vec<(Round, Author)>,
-    ) -> anyhow::Result<()> {
-        Ok(self.tx.unbounded_send(ordered_nodes)?)
+    ) {
+        self.tx.unbounded_send(ordered_nodes).unwrap()
```

Comment:
why unwrap instead of logging an error?

Reply:
I was worried about if one failed for some reason and we continue to send and succeed for some reason. It's simpler to hold the invariant imo